README.md

```diff
@@ -286,8 +286,9 @@ The memory usage limit is roughly estimated as `max_bytes * min_message_queue_si
 * `security_protocol` – Protocol used to communicate with brokers (`:ssl`)
 * `ssl_ca_location` – File or directory path to CA certificate(s) for verifying the broker's key
 * `ssl_crl_location` – Path to CRL for verifying broker's certificate validity
-* `ssl_keystore_location` – Path to client's keystore (PKCS#12) used for authentication
-* `ssl_keystore_password` – Client's keystore (PKCS#12) password
```

Comment:
We shouldn't remove existing options unless you're sure they don't work at all – others will rely on them.

Reply:
👍 added the new options without removing the old ones instead.